package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-social-icons",
+  "name": "@neesh775/react-social-icons",
```

Comment:
?

Reply:
The main changes should get implemented.... just that package.json thing is a bit....... messed up......

Reply:
sorry, new to the whole forking/pr'ing thing

Reply:
good now @ibnumalik

Reply:
good now @ibnumalik